src/modules/0.0.21/aws_cloudwatch/index.ts

```diff
@@ -11,40 +11,45 @@ class LogGroupMapper extends MapperBase<LogGroup> {
     Object.is(a.logGroupName, b.logGroupName) &&
     Object.is(a.logGroupArn, b.logGroupArn) &&
     Object.is(a.creationTime?.getTime(), b.creationTime?.getTime());
+  entityId = (e: LogGroup) => `${e.region}|${e.logGroupName}` ?? '';
```

Comment:
This will always produce a string of at least `|`, so the `?? ''` won't do anything.